.ddev/.setup/scripts/utils.sh

```
@@ -0,0 +1,563 @@
+    message magenta "|\t\t\t\t\t\t|"
+    message magenta "| \t\t     TYPO3 $VERSION     \t\t|"
+    message magenta "|\t\t\t\t\t\t|"
+    message magenta "-------------------------------------------------"
+}
+
+# Function to start the installation process for TYPO3.
+# It removes any existing files in the test directory for the specified version,
+# sets up the environment, creates symlinks for the main and additional extensions,
+# and sets up Composer for the TYPO3 installation.
+function install_start() {
+    rm -rf /var/www/html/.Build/$VERSION/*
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)